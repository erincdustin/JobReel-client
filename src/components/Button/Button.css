.Button {
  background: #56403e;
  border-radius: 10px;
  width: 10em;
  height: 2em;
  color: #eeebd9;
}

.Button:hover {
  transition: 0.3s;
  transform: scale(1.2);
  background: #3C824E;
  box-shadow: 0.5em 0.5em 0.2em rgb(48, 39, 39);
}

.Button:focus {
  outline: none;
}