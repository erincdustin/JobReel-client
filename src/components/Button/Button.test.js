import React from 'react';
import ReactDOM from 'react-dom'
import Button from './Button'
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Button smoke test', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Button />, div);
    ReactDOM.unmountComponentAtNode(div);
  })
})

describe('Button snapshot test', () => {
  it('renders the UI as expected', () => {
    const wrapper = shallow(<Button />)
    expect(toJson(wrapper)).toMatchSnapshot();
  });
})