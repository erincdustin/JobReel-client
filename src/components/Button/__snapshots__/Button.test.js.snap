// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button snapshot test renders the UI as expected 1`] = `
<button
  className="Button"
/>
`;
